### PR TITLE
[#69] Styling the breadcrumbs

### PR DESCRIPTION
**Pull request by nathan-schmidt-viget**
# Summary

Styles the breadcrumbs with TW

## Issues

* #69

## Testing Instructions

1. Make sure the breadcrumbs look like the screenshots. 

### Rank Math
Screenshot 2024-05-21 at 2 40 50 PM

### Default WP
Screenshot 2024-05-21 at 2 40 34 PM

### CODE REVIEW

File: wp-content/themes/wp-starter/src/styles/custom-blocks/breadcrumbs.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@layer components {
+	/* Rank Math breadcrumb styles */
+	.rank-math-breadcrumb {
+		p {
+			@apply flex flex-wrap gap-8;
+		}
+
+		a {
+			@apply text-sky-700 underline hover:no-underline focus:no-underline;
+		}
```
Comment (nathan-schmidt-viget) [inline wp-content/themes/wp-starter/src/styles/custom-blocks/breadcrumbs.css:10]: Not a fan of just tag elements but that is all Rank Math gives us 😞

File: wp-content/themes/wp-starter/blocks/breadcrumbs/block.json [hunk 2]
```diff
@@ -21,12 +21,11 @@
     "layout": {
       "default": {
         "type": "constrained",
-        "justifyContent": "center"
+        "justifyContent": "left"
```
Comment (bd-viget) [inline wp-content/themes/wp-starter/blocks/breadcrumbs/block.json:24]: 👍

File: wp-content/themes/wp-starter/blocks/page-header/patterns/page-header-inner-blocks.php
Comment (bd-viget) [inline wp-content/themes/wp-starter/blocks/page-header/patterns/page-header-inner-blocks.php:1]: I assume these changes came from your IDE? I would be careful modifying the block template formatting from what is copied/pasted from the editor. There's a risk of getting a "Block appears to be broken" error.

**Review by bd-viget — APPROVED** [review]
👍 LGTM!

File: wp-content/themes/wp-starter/blocks/page-header/patterns/page-header-inner-blocks.php
Comment (nathan-schmidt-viget) [reply wp-content/themes/wp-starter/blocks/page-header/patterns/page-header-inner-blocks.php:1]: Ya so I have a formatted in my VS code for these files which makes them easier to read. And I used it on all of the GB parts and patterns. It only changes the lines and indents as anything else (quotes, spacing, linewrap) can and will mess up the block.

**Pull request merged**